Headers/DebugServer2/GDBRemote/Types.h

```diff
@@ -208,6 +208,30 @@ struct ProgramResult {
 
   std::string encode() const;
 };
+
+struct ptidIterator {
+  std::vector<ProcessThreadId> ptids;
+  size_t index = 0;
+
+  inline void clear() {
+    index = 0;
+    ptids.clear();
+  }
+
+  inline void addPid(ProcessId pid) { ptids.push_back(ProcessThreadId(pid)); }
+
+  inline void reset(std::vector<ThreadId> const &tids) {
+    clear();
+    for (auto &tid : tids)
```

Comment:
for uniformity I think we should always use { } 
you can also use emplace_back in this case, avoiding some copies